### DISCUSSION TITLE
Kong 2.8.1 :  could not reschedule DNS resolver timer

### DISCUSSION THREAD

**Discussion by sunny-deo-incontact** [Kong/kong#9688] [Help] [answered]
Hi All,

We are getting below error with Kong 2.8.1
![image](https://user-images.githubusercontent.com/94824593/200026600-0f65d40f-3351-43f4-bf96-e7d38b239d0e.png)

After this all our nginx worker process gets killed and are respawned. 

Any idea why this issue might come?  

Note: Same error is not coming in 2.0.3 version

**Comment by ADD-SP** [comment, marked as answer]
I think this is because the process exits when the DNS timer starts, at which point this log is expected. 

But in this case this log level is too high.
